framework/src/processing/lib/spark-job/pyspark-application-package.ts

```diff
@@ -160,12 +171,12 @@ export class PySparkApplicationPackage extends TrackedConstruct {
           memoryLimit: 512,
```

Comment:
We need to expose these the memotyLimit and ephemeralStorage as Props.

Reply:
done